### ISSUE TITLE
Accessibility - Web - Application is Missing Focus for the Section Show/Hide

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Ilab-do-good#123]
While reviewing the application, I noticed when I was tabbing on the "Step 3: Develop a Code of Conduct" screen for example in IE 11 I could not tell when I was on the show and hide  in the "Learning Objectives", "Key Terms", and "Further Resources" sections of the page. There was no visual focus on these items.

I'm assuming the issue the same thing happens to Firefox, Chrome, and Safari but some of those browsers aren't loading the application or don't even tab into the main content section of the page. Also, assuming there are similar issues on other pages.

I checked all three languages (English, Spanish, and French) and the same things occur for them as well.

**Comment by sanganalakshay** [comment]
should be fixed in the latest push

**Comment by jfc3-dol** [comment]
Re-opening issue because the focus is not showing to Firefox after making an update to show the link focus in Firefox.

**Comment by sanganalakshay** [comment]
@jfc3-dol 
I am able to see dotted lines as focus ... and it focuses on links as well as expandable tabs 

![image](https://user-images.githubusercontent.com/15368456/43854319-90d9300c-9b10-11e8-8ce3-1f4f5f47119f.png)

**Comment by jfc3-dol** [comment]
The application 's still having issues depending on the browser  (IE 11, Firefox, and Chrome) after refreshing the page a few times. 

Will reboot machine tomorrow to see if that helps so I can attempt get through other issues today before I head home.
